### ISSUE TITLE
My bios is... broken?

### ISSUE THREAD

**Issue by MatiLol251** [melonDS-emu/melonDS#1682]
i installed some dsi ware games, but when i try to boot the NAND, it appears the next error message:
![image](https://github.com/melonDS-emu/melonDS/assets/106706551/691ad6f7-5b29-44cb-a56a-5b1b2c04f7e1)

**Comment by CasualPokePlayer** [comment]
Then it indicates one of your bios/firmware/NAND is just bad. There's nothing melonDS can do, get better dumps and try again.

**Comment by RSDuck** [comment]
did it work before you installed the DSiware?

**Comment by MatiLol251** [comment]
> did it work before you installed the DSiware?

Yes, it worked. I even installed some DSiWare software. But i guess i installed a lot of games without testing them.

**Comment by RSDuck** [comment]
is it possible that your NAND is full? Somebody else had that issue recently.

**Comment by Peter0x44** [comment]
#1285 is the same thing, with the full NAND
